### PR TITLE
Profiles for new LTS JDK

### PR DESCRIPTION
**Pull request by tkyc**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -46,6 +46,19 @@ if (!hasProperty('buildProfile') || (hasProperty('buildProfile') && buildProfile
 	targetCompatibility = 22
 }
 
+if (!hasProperty('buildProfile') || (hasProperty('buildProfile') && buildProfile == "jre21")) {
```
Comment (Jeffery-Wasty) [inline build.gradle:49]:
```suggestion
if (hasProperty('buildProfile') && buildProfile == "jre21") {
```

We want default to be 22.

**Review by Jeffery-Wasty — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (Jeffery-Wasty) [reply build.gradle:49]: Never mind, if there is no build profile it will build with 22, no need to remove.

Comment (barryw-mssql) [inline build.gradle:49]: Please correct me if I am wrong but isn't (!hasProperty('buildProfile') || (hasProperty('buildProfile') always true?

Comment (barryw-mssql) [reply build.gradle:49]: Sorry ... missed the order of brackets. Never mind

**Review by barryw-mssql — APPROVED** [review]

**Pull request merged**